### PR TITLE
Documentation for summer workshop

### PR DESCRIPTION
**Pull request by mfsilva22**
This can be added to the website, or just used for the workshop this Summer. My initial plan is:

- once we have a list of the people interested in participating in the workshop, we can:

1. create a course repo if they are already invested in using PL
2. add them to pl-train101, if they want to just try using PL during the workshop

- participants should read this first "get started tutorial" before coming to day one. Or we can just go over the steps from the tutorial.

- day 2 of the workshop: go over more details, such as exams vs homework assessment, time limit, password, zones, point system, etc. This can be the first 30 min. The other half can be for questions. I can create a documentation for that second part as well.

day 3-5: focus on question creation. we can ask participants to submit their questions, and we just help with review/feedback.

### CODE REVIEW

File: docs/getStarted.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+![](start-guide/change-example-course.png)
+
+You will find a variety of questions in the example course. From the `Assessments` tab, you can find the questions organized by type. This is probably your best starting point when creating questions for the first time.
+
+Let's see how you can copy one of the example questions to your own course:
+
+* click on the assessment `HW1: Template homework - copy this to start a new HW`.
+
+* click on the question `Template pl-integer-input: randomized input parameters` (QID: template/integerInput).
```
Comment (nicknytko) [inline docs/getStarted.md:221]:
```suggestion
* click on the question `Template pl-integer-input: randomized input parameters` (QID: `template/integerInput`).
```

File: docs/getStarted.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+]
+```
+
+* click `Save and sync`.
+
+Read the [assessment documentation](https://prairielearn.readthedocs.io/en/latest/assessment/) to learn more about the different assessment configurations available.
+
+Learn more:
+[Quick reference guide about question structure and PrairieLearn elements](https://coatless.github.io/pl-cheatsheets/pdfs/prairielearn-authoring-cheatsheet.pdf)
+[Detailed list of PrairieLearn elements](https://prairielearn.readthedocs.io/en/latest/elements/)
```
Comment (nicknytko) [inline docs/getStarted.md]:
```suggestion
- [Quick reference guide about question structure and PrairieLearn elements](https://coatless.github.io/pl-cheatsheets/pdfs/prairielearn-authoring-cheatsheet.pdf)
- [Detailed list of PrairieLearn elements](https://prairielearn.readthedocs.io/en/latest/elements/)
```

Lets add bullet points?

File: docs/requestCourse.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Request your course space
+
+To request your PrairieLearn course space, you should first join our Slack channel:
```
Comment (nicknytko) [inline docs/requestCourse.md:3]:
```suggestion
To request your PrairieLearn course space, you should first join the PrairieLearn Slack channel:
```

File: docs/requestCourse.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+[https://go.illinois.edu/joinplslack](https://go.illinois.edu/joinplslack)
+
+In the `#pl-help` channel, please provide the following information:
```
Comment (nicknytko) [inline docs/requestCourse.md:7]: Do we want to create a `#course-requests` or similar channel?

File: mkdocs.yml [hunk 5]
```diff
@@ -30,6 +30,9 @@ pages:
   - 'How to use PL to learn student names': 'howToLearnNames.md'
   - 'FAQ': 'faq.md'
   - 'API': 'api.md'
+- Getting Started:
+  - 'Requesting your course space': 'requestCourse.md'
+  - 'Creating content in the browser': 'getStarted.md'
```
Comment (mwest1066) [inline mkdocs.yml:35]: Can we move this up to the top?

Comment (mfsilva22) [reply mkdocs.yml]: I am so puzzled... I made these changes yesterday, I am not sure how it did not get added. I changed it again now.

**Pull request merged**